react_render/django/render.py

```diff
@@ -41,6 +41,7 @@ def render_props(self):
 
     def render_props_b64(self):
         if self.props:
+            self.json_encoder = json.JSONEncoder(separators=(',', ':')).encode
             encoded = base64.b64encode(self.json_encoder(self.props).encode("utf-8")).decode('utf-8')
```

Comment:
This has an issue in the current form it does not work 
work around:
`json.dumps(self.props, cls=self.json_encoder.__class__)`